packages/kotti-table/src/logic/select.js

```diff
@@ -61,14 +61,22 @@ export const getters = {
 			: row[state.rowKey]
 	},
 	isSelected(state, row) {
-		return (state.selection || []).indexOf(row) > -1
+		return (
+			(state.selection.map(e => JSON.stringify(e)) || []).indexOf(
```

Comment:
https://www.npmjs.com/package/deep-eql or https://www.npmjs.com/package/shallow-equal

Reply:
@carsoli did you address this?

@FlorianWendelborn would deep equal windup interacting with Vue's proxies?
I think stringily does the job of deep equal as far as the kind of props used in the columns are concerned as far as we need it too, if we're talking performance we may be ok with shallow equal